RELEASE.md

```diff
@@ -0,0 +1,17 @@
+# Publishing a New Release
+*This page contains instructions for admins of this project to release a new version.*
+
+## Run Makefile
+Run `$ make publish VERSION_NUMBER` at the root directory of needle (where the `Makefile` is located)
+
+For example:
+```
+$ make publish 0.13.0
+```
+
+This runs the steps specified in the `Makefile`.
+
+The `Makefile` does support failure recovering yet. If a certain step fails, you might want to figure out which step failed, and start manually run the remaining steps.
```

Comment:
you mean 'does not support'? 

s/recovering/recovery 

remove 'start'

Reply:
Nice, thanks for catching those errors. Fixed